src/github.com/getlantern/detour/whitelist.go

```diff
@@ -19,7 +19,9 @@ var (
 func AddToWl(addr string, permanent bool) {
 	muWhitelist.Lock()
 	defer muWhitelist.Unlock()
-	whitelist[addr] = wlEntry{permanent}
+	if add != "" {
```

Comment:
> `addr` will never be "" except passed in intentionally.

That's what I am trying to prevent, with this check the function will take care for itself, instead of relying on the context.